modules/serverless-rn-1-36-0.adoc

```diff
@@ -0,0 +1,107 @@
+
+* The KafkaSource API has now been promoted to version `v1`, signaling its stability and readiness for production use.
+
+* {ServerlessProductName} now supports deployment on ARM architecture as a Generally Available (GA) feature.
+
+* The `kn event` plugin is now available as a Generally Available (GA) feature. This plugin allows you to send events directly from the command line to various destinations, streamlining event-driven application development and testing workflows.
+
+* When starting a workflow via HTTP, you can now include additional properties alongside the `workflowdata` field in the request body. These extra fields are ignored by the runtime but are available in the Data Index as process variables as shown in the following example:
++
+[source,json]
+----
+{"workflowdata": {"name": "John"}, "groupKey": "follower"}
+----
+
+* It is now possible to filter workflow instances by the content of workflow variables using GraphQL queries on `ProcessInstances.variables`. For example, the following query retrieves process instances where the `language` field in `workflowdata` equals `Spanish`:
++
+[source,terminal]
+----
+ProcessInstances (where:{variables:{workflowdata:{language:{equal:Spanish}}}}) {
+    variables,
+    state,
+    lastUpdate,
+    nodes {
+      name
+    }
+}
+----
+
+* {ServerlessLogicProductName} Data Index now supports filtering queries using workflow definition metadata. Additionally, Data Index has been updated to consume deploy and undeploy events, improving the accuracy and traceability of workflow state. 
+
+* {ServerlessLogicOperatorName} now emits events to the Data Index to indicate when a workflow definition becomes available or unavailable. These deploy and undeploy events are sent automatically when the Data Index is present in the same namespace. 
+
+[id="fixed-issues-1-36-0_{context}"]
+== Fixed issues
+
+* Previously, the Knative Kafka dispatcher could stop consuming events if a Kafka consumer group rebalance occurred while a sink was processing events out of order. This behavior triggered errors such as:
+
+** `SEVERE: Unhandled exception`
+** `java.lang.IndexOutOfBoundsException: bitIndex < 0`
+** Repeated logs like `Request joining group due to: group is already rebalancing`
+
+This issue is now fixed. The dispatcher correctly handles out-of-order event consumption during rebalancing and continues processing events without interruption.
+
+* Previously, a KafkaSource remained in a `Ready` state even when `KafkaSource.spec.net.tls.key` failed to load due to the use of unsupported TLS certificates in PKCS #1 format. This issue is now fixed. An appropriate error is now reported when attempting to create a KafkaBroker, KafkaChannel, KafkaSource, or KafkaSink using TLS certificates in an unsupported format.
```

Comment:
Should the following instances be in back ticks? +1 to ticks if these are CR names
- KafkaBroker, KafkaChannel, KafkaSource, KafkaSink

Reply:
Yes, you are right! They should be in back ticks.